scheduled-publication-workflow/scheduled-publication-workflow-server/src/main/resources/com/coremedia/blueprint/scheduledpublication/studio-scheduled-publication.xml

```diff
@@ -143,19 +143,15 @@
       </Assignment>
     </UserTask>
 
-    <Variable name="finishTaskRetryTimer" type="Timer"/>
-    <TimerHandler class="RetryTaskTimerHandler" task="Finish" timerName="finishTaskRetryTimer"/>
-
     <AutomatedTask name="Finish" final="true">
       <Action class="AssignVariable" resultVariable="changeSetLockedInStudio">
         <Boolean value="false"/>
       </Action>
-      <Action class="AssignVariable" resultVariable="finishTaskRetryTimer">
-        <Timer value="30"/>
-      </Action>
-      <Action class="EnableTimer" timerVariable="finishTaskRetryTimer"/>
+      <Action class="ArchiveProcess" maxProcessesPerUser="100"/>
```

Comment:
You have re-introduced the deprecated ArchiveProcesss action here. It doesn't do anything and might be deleted in the future. You should remove that line from all workflow definitions.